.env.sample

```diff
@@ -1,4 +1,5 @@
 DATABASE_CLIENT=
 DATABASE_URL=
 TEST_DATABASE_CLIENT=
-TEST_DATABASE_URL=
\ No newline at end of file
+TEST_DATABASE_URL=
+SENDGRID_API_KEY=
```

Comment:
Please can you add this to the readme.md file ? some sort of documentation so others can know how to use. You could also share your API key on slack :)

Reply:
readme.md file updated and I have shared the API key credential with team members on slack